spacialgaze-plugins/economy.js

```diff
@@ -235,31 +273,16 @@ exports.commands = {
 	'!richestuser': true,
 	richestusers: 'richestuser',
 	richestuser: function (target, room, user) {
-		if (!target) target = 10;
+		if (!target) target = 100;
 		target = Number(target);
-		if (isNaN(target)) target = 10;
+		if (isNaN(target)) target = 100;
 		if (!this.runBroadcast()) return;
-		if (this.broadcasting && target > 10) target = 10; // limit to 10 while broadcasting
-		if (target > 500) target = 500;
-
-		let self = this;
-
-		function showResults(rows) {
-			let output = '<table border="1" cellspacing ="0" cellpadding="3"><tr><th>Rank</th><th>Name</th><th>' + currencyPlural + '</th></tr>';
-			let count = 1;
-			for (let u in rows) {
-				if (rows[u].amount < 1) continue;
-				output += '<tr><td>' + count + '</td><td>' + SG.nameColor(rows[u].name, true) + '</td><td>' + rows[u].amount + '</td></tr>';
-				count++;
-			}
-			self.sendReplyBox(output);
-			if (room) room.update();
-		}
-		let obj = Db.currency.keys().map(function (name) {return {name: name, amount: Db.currency.get(name)};});
-		let results = obj.sort(function (a, b) {
-			return b.amount - a.amount;
+		let keys = Db.currency.keys().map(function (name) {
+			return {name: name, money: Db.currency.get(name)};
```

Comment:
The callback can be simplified to `name => ({name, money: Db.currency.get(name)})`